kubernetes/module_defaults

```diff
@@ -17,4 +17,5 @@ MOD_PARAMS[__NUM_K8S_WORKER_UNITS__]=2
 MOD_PARAMS[__NUM_K8S_LB_UNITS__]=1
 MOD_PARAMS[__ETCD_SNAP_CHANNEL__]='latest/stable'
 MOD_PARAMS[__CONTAINER_RUNTIME__]='containerd'
-
+MOD_PARAMS[__CONTAINERD_PROXY__]='http://squid.internal:3128'
```

Comment:
rather than setting proxy by default lets leave empty and populate if we are running in prodstack which we can detect by doing e.g. ```if `juju show-cloud prodstack &>/dev/null`; then ...```

Reply:
Implemented this